pkg/oc/cli/cmd/importimage.go

```diff
@@ -299,42 +298,28 @@ func (e importError) Error() string {
 }
 
 func (o *ImportImageOptions) waitForImport(resourceVersion string) (*imageapi.ImageStream, error) {
-	streamWatch, err := o.isClient.Watch(metav1.ListOptions{FieldSelector: fields.OneTermEqualSelector("metadata.name", o.Name).String(), ResourceVersion: resourceVersion})
-	if err != nil {
-		return nil, err
-	}
-	defer streamWatch.Stop()
-
-	for {
-		select {
-		case event, ok := <-streamWatch.ResultChan():
-			if !ok {
-				return nil, fmt.Errorf("image stream watch ended prematurely")
+	var is *imageapi.ImageStream
+	err := wait.Poll(1*time.Second, 60*time.Second, func() (bool, error) {
```

Comment:
`wait.PollImmediate` ?

Reply:
during testing, it usually polled ~3 times, would we gain anything with `wait.PollImmediate`?

Reply:
in case the server is fast, we don't have to wait 1s at least